openmls/src/key_packages/mod.rs

```diff
@@ -347,13 +347,11 @@ impl KeyPackage {
         }
 
         // Verify the signature on this key package.
-        <Self as Verifiable>::verify_no_out(
-            self,
-            backend,
-            self.leaf_node().signature_key(),
-            self.leaf_node().credential().signature_scheme(),
-        )
-        .map_err(|_| {
+        let pk = OpenMlsSignaturePublicKey::from_signature_key(
+            self.leaf_node().signature_key().clone(),
```

Comment:
Maybe not for this PR, but we should really try to use OpenMlsSignaturePublicKey everywhere.